crates/opencascade-sys/Cargo.toml

```diff
@@ -13,7 +13,7 @@ cxx = "1"
 [build-dependencies]
 cmake = "0.1"
 cxx-build = "1"
-occt-sys = { version = "0.5.1", optional = true }
+occt-sys = { path = "../occt-sys", optional = true }
```

Comment:
@bschwind not sure about this one here

Reply:
Ah, I think we intentionally don't use the local dependency in order for our CI setup to cache it properly. See [this comment](https://github.com/bschwind/opencascade-rs/pull/91#issuecomment-1641572098).

Though we should revisit if that assumption still holds.

Reply:
so I use the 0.5.1 version?

Reply:
I think we'll have to do it in two parts:

* I'll review the occt-sys changes and once we agree on those we can publish `0.5.2` or `0.6` depending on if it's a breaking change or not
* We can then make the (somewhat trivial) `opencascade-sys` change separately

Sorry for the trouble on that, it's mostly trying to optimize for caching and faster CI build times. I'm also working on getting a windows dev machine set up to more easily be able to test this.

Reply:
Seems reasonable.

Reply:
done. Then there will be a follow-up once if released.